src/_gettsim_tests/test_data/wohngeld/2006/hh_id_15.yaml

```diff
@@ -312,7 +312,16 @@ inputs:
       - true
       - true
       - true
-  assumed: {}
+  assumed:
```

Comment:
Why not make it the same as in the next few tests, i.e., adding einstandspartner etc? No big deal, but might make people wonder who look at these one after the other.